harness/determined/common/__init__.py

```diff
@@ -4,5 +4,6 @@
     # Inexplicably, sometimes ruamel.yaml is pacakged as ruamel_yaml instead.
     import ruamel_yaml as yaml  # type: ignore
 
-from . import api, check, constants, context, requests, storage, util
-from ._logging import set_logger
+from determined.common import util
```

Comment:
is this separate from the line below to make sure it is imported before all the other things (`api`, `check`, `constants` etc.) ?

Reply:
yes, since `common.api._session` imports `common.util`.

Long-term, I want to redo our internal code layout to reflect the more opinionated public api layout that we have.  I'm thinking something like:

```python
determined.internal                 # no symbols here, maybe?
determined.internal.bindings
determined.internal.constants       # only one constants module!
determined.internal.util            # only one util module!
determined.internal.request         # unroll common.api
determined.internal.authentication  # ...

# OR maybe don't unroll common.api, but eliminate the submodules from import paths
determined.internal.api._bindings   # exposed as api.*
determined.internal.api._request    # exposed as api.*
# ...
```

This would also make it crystal clear to users what counts as internal and our own import paths would be a lot less confusing.